cruise.umple/test/cruise/umple/implementation/php/BasicPostcondition1.php.txt

```diff
@@ -1,6 +1,6 @@
 <?php
 /*PLEASE DO NOT EDIT THIS CODE*/
-/*This code was generated using the UMPLE @UMPLE_VERSION@ modeling language!*/
+/*This code was generated using the UMPLE 1.25.0-963d2bd modeling language!*/
```

Comment:
I generated this code via my build at `/dist/umple-*`. Is there another way in which I should be creating the `*.txt` test files? Or should I just change that commented line in the generated files?